### ISSUE TITLE
[New feature] Ban backend user

### ISSUE THREAD

**Issue by pvullioud** [octobercms/october#3527]
##### Expected behavior
Allow to ban some backend user but keep their information. This is useful when someone leave a team. We can keep the information of the author (in a blog post) or for other relations

##### Actual behavior
Nothing for the backend, Rainlab.User provide this functionality for frontend

I'm ready to code it. I can do it as a plugin or in core.

**Comment by ChVuagniaux** [comment]
All is already ready in October core with the Throttle management.

The only missing thing is an interface for manage that.

For e.g. the plugin we have made for implement that :
https://github.com/inetis-ch/oc-banBackendUsers-plugin

**Comment by bhargavy** [comment]
Similar Issue - #3418 Backend users Activation and Deactivation (Suspended) Suggestion

**Comment by pvullioud** [comment]
The plugin exist on github: https://github.com/inetis-ch/oc-banBackendUsers-plugin
@LukeTowers should we add this feature into the core ?

**Comment by Samuell1** [comment]
@pvullioud i think better option will be to allow backenduser to be disabled/enabled.

**Comment by pvullioud** [comment]
I agree with @bennothommo . we will release the plugin on the market place. 
I this feature is added later in the core, we can delete it.
